doc/usecases/layers/shadows_layer.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Shadows Layer
+
+## Summary
+
+- **Scope:** Shadow Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can add, edit, move and delete shadow indicators in their plan using the wind layer
```

Comment:
- This is again a zones map (big brush showing how shady it is at which places)
- Copy&paste error: wind -> shadow